doc/api/net.md

```diff
@@ -544,40 +729,41 @@ If `data` is specified, it is equivalent to calling
 added: v0.9.6
 -->
 
-The string representation of the local IP address the remote client is
-connecting on. For example, if you are listening on `'0.0.0.0'` and the
-client connects on `'192.168.1.1'`, the value would be `'192.168.1.1'`.
+Provides the string representation of the local IP address the remote peer is
+connecting on. For example, if the `net.Socket` is listening on `'0.0.0.0'` and
+the peer connects on `'192.168.1.1'`, the value would be `'192.168.1.1'`.
 
 ### socket.localPort
 <!-- YAML
 added: v0.9.6
 -->
 
-The numeric representation of the local port. For example,
-`80` or `21`.
+Provides the numeric representation of the local port. For example, `80` or
+`21`.
 
 ### socket.pause()
 
-Pauses the reading of data. That is, [`'data'`][] events will not be emitted.
-Useful to throttle back an upload.
+The `socket.pause()` method pauses the reading of data. That is, [`'data'`][]
+events will not be emitted. See the [Readable][] documentation for more
+information.
 
 ### socket.ref()
 <!-- YAML
 added: v0.9.1
 -->
 
-Opposite of `unref`, calling `ref` on a previously `unref`d socket will *not*
-let the program exit if it's the only socket left (the default behavior). If
-the socket is `ref`d calling `ref` again will have no effect.
+Calling the `socket.ref()` method will cause the Node.js event loop to continue
```

Comment:
as commented above, doesn't describe that the default is refed. This exists to undo a previous `.unref()`